tools/example-module/example-module.template

```diff
@@ -1,43 +1,20 @@
 /**
  ******************************************************************************
  * DO NOT MANUALLY EDIT THIS FILE. THIS FILE IS AUTOMATICALLY GENERATED.
- *
- * IMPORTANT: Due to a bug with the bazel ng_module rule, this file is now
- * checked into the repository. Please rebuild the example module by running
- * the following command: "yarn gulp build-examples-module"
- *
- * BUG: https://github.com/angular/angular/issues/30259
  ******************************************************************************/
 
-import {NgModule} from '@angular/core';
-import {MAT_FORM_FIELD_DEFAULT_OPTIONS} from '@angular/material/form-field';
-
-${exampleImports}
-
 export interface LiveExample {
   title: string;
-  component: any;
+  component: string;
+  path: string;
   additionalFiles?: string[];
   selectorName?: string;
+  module: ModuleInfo;
 }
 
-export const EXAMPLE_COMPONENTS: {[key: string]: LiveExample} = ${exampleComponents};
-
-export const EXAMPLE_MODULES = ${exampleModules};
-
-export const EXAMPLE_LIST = [${exampleList}];
-
-// Default MatFormField appearance to 'fill' as that is the new recommended approach and the
-// `legacy` and `standard` appearances are scheduled for deprecation in version 10.
-@NgModule({
-  imports: EXAMPLE_MODULES,
-  exports: EXAMPLE_MODULES,
-  providers: [
-    { provide: MAT_FORM_FIELD_DEFAULT_OPTIONS, useValue: { appearance: 'fill' } },
-  ],
-})
-export class ExampleModule { }
+export interface ModuleInfo {
+  name: string;
+  path: string;
+}
```

Comment:
Could you add some comments for this interface and its members?

Reply:
Added descriptions for the properties. Also cleaned up a few more things. Most of this code is pretty "legacy"

Reply:
The thing I wasn't clear about was whether this was an `NgModule` or a JavaScript module; I should have been more specific

Reply:
oh I see. My bad. Renamed the interface to `NgModuleInfo`.